client/src/components/campaign/round/round.html

```diff
@@ -0,0 +1,135 @@
+        <span>{{ ::$ctrl.round.name }}</span>
+        <span class="muted">{{ ::$ctrl.round.vote_method }} · {{ $ctrl.round.status }}</span>
+      </div>
+      <md-button
+          ng-click="$ctrl.editRound()"
+          ng-disabled="$ctrl.round.status !== 'paused'">
+        <md-icon>settings</md-icon>
+        Edit Round
+      </md-button>
+    </h2>
+    <md-card class="round__description" flex>
+      <md-card-content>
+        <div layout="row" layout-align="center stretch">
+          <div class="round__directions" flex="60">
+            <h4>Deadline Date</h4>
+            <p>{{ ::$ctrl.round.deadline_date | date: 'd MMM yyyy' }} · <span class="muted">{{ ::$ctrl.round.deadline_date | fromNow }}</span></p>
+            <h4>Directions</h4>
+            <p>{{ ::$ctrl.round.directions }}<span class="muted" ng-if="!$ctrl.round.directions">No directions provided.</span></p>
+          </div>
+          <div flex></div>
+          <div class="round__settings" flex="30">
+            <h4>Round File Settings</h4>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.config">
+                <span>{{ ::name | translate }}</span>
+                <span>{{ ::value | json}}</span>
+              </span>
+            </p>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.details.counts">
+                <span>{{ ::("round." + name) | translate }}</span>
+                <span>{{ ::value | json}}</span>
+              </span>
+            </p>
+          </div>
+        </div>
+        
+        <div class="round__jurors"
+            layout="row" layout-align="center stretch">
+          <div class="round__directions" flex>
+            <h4>Quorum</h4>
+            <p>{{ ::$ctrl.round.quorum }} jurors per photo</p>
+            <h4>Jurors</h4>
+            <div class="round__juror-container"
+                layout="column" layout-align="center stretch"
+                ng-repeat="juror in $ctrl.round.jurors | orderBy:'username'">
+              <span class="round__juror"
+                  ng-class="juror.is_active ? 'round__juror--active' : 'round__juror--inactive'">
+                <span class="juror__avatar" mont-avatar="{{ ::juror.username }}">
+                  {{ ::juror.username[0].toUpperCase() }}
+                </span>
+                {{ ::juror.username }}
+              </span>
+              <div class="juror__stats">
+                <span class="juror__percentage"
+                    ng-if="juror.stats.total_tasks">
+                  {{ ::(100 - juror.stats.percent_tasks_open) | number : 1 }}%
+                </span>
+                <span class="juror__percentage"
+                    ng-if="!juror.stats.total_tasks">
+                  N/A
```

Comment:
If this is N/A, it means that tasks are not yet assigned. We should probably not show "0 out of 0" tasks for the user, since that's a bit confusing. Instead, let's say: "Activate the round to assign tasks and start voting"